AMW_rest/src/main/java/ch/mobi/itc/mobiliar/rest/deployments/DeploymentsRest.java

```diff
@@ -363,6 +367,32 @@ public Response getAllDeploymentParameterKeys() {
         return Response.status(Status.OK).entity(deploymentParameters).build();
     }
 
+    @POST
+    @Path("/deploymentParameterKeys/")
+    @ApiOperation(value = "Adds one parameter key")
+    public Response addOneParameterKey(DeploymentParameterDTO deploymentParameterDTO) {
+        Key keyToBeAdded = new Key(deploymentParameterDTO.getKey());
+        keyRepository.createDeployParameterKey(keyToBeAdded);
```

Comment:
Direct access of the repository is not recommended. Please use the according function in the boundary package